### PR TITLE
[MM-44649] - Add CTA to contact sales in the enterprise card on the pricing modal

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR adds `Contact Sales for a quote` CTA and `Notify Admin` CTA for non-admin users

#### Ticket Link
[MM-44649](https://mattermost.atlassian.net/browse/MM-44649)
[MM-43648](https://mattermost.atlassian.net/browse/MM-43648)

#### Screenshots

Screenshot 2022-05-31 at 14 26 03

#### Release Note

```release-note
none
```

### CODE REVIEW

File: components/pricing_modal/pricing_modal.scss [hunk 1]
```diff
@@ -245,6 +249,7 @@
                     &.grayed {
                         background: rgba(var(--center-channel-color-rgb), 0.08);
                         color: rgba(var(--center-channel-color-rgb), 0.32);
+                        cursor: not-allowed;
```
Comment (neallred) [inline components/pricing_modal/pricing_modal.scss:252]: :+1:

**Review by marianunez — COMMENTED** [review]
Could we update the cypress test to include the new cases for end users in the pricing modal?

File: components/pricing_modal/content.tsx [hunk 2]
```diff
@@ -241,9 +248,7 @@ function Content(props: ContentProps) {
                             action: () => {}, // noop until we support downgrade
                             text: formatMessage({id: 'pricing_modal.btn.upgrade', defaultMessage: 'Upgrade'}),
                             disabled: true, // disabled until we have functionality to downgrade
-                            customClass: ButtonCustomiserClasses.grayed,
                         }}
-                        planDisclaimer={formatMessage({id: 'pricing_modal.planDisclaimer.starter', defaultMessage: 'This plan has data restrictions.'})}
```
[the remark this answers is not in the document]
Comment (AGMETEOR) [reply components/pricing_modal/content.tsx:246]: This CTA is going to be added in another PR here https://github.com/mattermost/mattermost-webapp/pull/10392

File: components/pricing_modal/content.tsx [hunk 3]
```diff
@@ -275,9 +280,11 @@ function Content(props: ContentProps) {
                                 formatMessage({id: 'pricing_modal.extra_briefing.professional.guestAccess', defaultMessage: 'Guest access with MFA enforcement'}),
                             ],
                         }}
+                        planExtraInformation={isAdmin ? undefined : <NotifyAdminCTA/>}
```
[the remark this answers is not in the document]
Comment (AGMETEOR) [reply components/pricing_modal/content.tsx:283]: Yes sure

**Review by marianunez — APPROVED** [review]
Thanks @AGMETEOR! Could we update the cypress test to include the new cases for end users in the pricing modal?

**Review by annelieseklein — APPROVED** [review]
LGTM!

**Comment by AGMETEOR** [comment]
> https://mattermost-webapp-pr-10416.test.mattermost.cloud

Added cypress tests. Non admin users can't open this modal yet, so any test cases will be added here https://github.com/mattermost/mattermost-webapp/pull/10437

**Review by stevemudie — APPROVED** [review]
Worked as expected! All set @AGMETEOR

**Pull request merged**